data_dir/nemesis.yml

```diff
@@ -206,10 +228,7 @@
   - run_with_gemini = False
   - kubernetes = True
   - limited = True
-- disrupt_nodetool_refresh:
```

Comment:
@yarongilor why this nemesis got removed? (same for enospc)

Reply:
@soyacz , because it was erroneously duplicated.